test/ussd_covid19_triage.test.js

```diff
@@ -152,6 +185,41 @@ describe("ussd_covid19_triage app", function () {
                 })
                 .run();
         });
+        it("should repeat question on invalid input confirmed contact", function () {
+            return tester
+                .setup.user.state("state_welcome")
+                .setup.user.answer("confirmed_contact", true)
+                .input("A")
+                .check.interaction({
+                    state: "state_welcome",
+                    reply: [
+                        "The Dept of Health: you have been in contact with someone who has " +
+                        "COVID-19. Isolate for 10 days & answer these questions.",
+                        "",
+                        "Reply",
+                        "1. START"
+                    ].join("\n"),
+                    char_limit: 140
```

Comment:
not sure why you have a limit of 140 chars' where in all other places its 160

Reply:
It's the first screen of the session. Some MNOs have a lower limit for the first screen you show a user vs the rest of the screens.